webpack.config.js

```diff
@@ -71,13 +73,28 @@ const config = {
           },
         ],
       },
-      {
-        test: /\.json$/,
-        exclude: /node_modules/,
-        loader: 'json-loader',
-      },
     ],
   },
+  optimization: {
+    runtimeChunk: {
+      name: 'runtime',
+    },
+    splitChunks: {
+      cacheGroups: {
+        runtimeChunk: 'single',
+        // vendor: {
+        //   name: 'vendor',
+        //   chunks: 'initial',
+        //   test: /[\\/]node_modules[\\/]/,
+        // },
+        default: {
+          name: 'common',
+          chunks: 'all',
+          minChunks: 2,
+        },
```

Comment:
Been poking at this on and off in Force but the bundle sizes are a bit larger for some reason. Need to return to it at some point.